tests/snuba/test_deletes.py

```diff
@@ -0,0 +1,77 @@
+import time
+from datetime import datetime, timedelta
+from uuid import uuid4
+
+import pytest
+from snuba_sdk import DeleteQuery, Request
+
+from sentry.snuba.dataset import Dataset
+from sentry.testutils.cases import SnubaTestCase, TestCase
+from sentry.utils import snuba
+
+
+class SnubaTest(TestCase, SnubaTestCase):
+    @pytest.mark.xfail
+    def test_basic(self) -> None:
+        # insert a new issue
+        now = datetime.now()
+        id = uuid4()
+        issue = (
+            2,
+            "insert",
+            {
+                "group_id": 1,
+                "message": "message",
+                "platform": "python",
+                "primary_hash": "1" * 32,
+                "event_id": "a" * 32,
+                "project_id": self.project.id,
+                "datetime": now.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+                "data": {"received": time.mktime(now.timetuple())},
+                "occurrence_data": {
+                    "detection_time": time.mktime(now.timetuple()),
+                    "fingerprint": ["hi"],
+                    "issue_title": "myissue",
+                    "id": id,
+                    "type": 1,
+                },
+                "organization_id": 6,
+            },
+        )
+        self.store_issues([issue])
+
+        # make sure its there
+        assert snuba.query(
+            dataset=Dataset.IssuePlatform,
+            start=now - timedelta(days=1),
+            end=now + timedelta(days=1),
+            groupby=["project_id"],
+            filter_keys={"project_id": [self.project.id]},
+            referrer="testing.test",
+            tenant_ids={"referrer": "testing.test", "organization_id": 1},
+        ) == {self.project.id: 1}
+
+        # delete it
+        req = Request(
+            dataset="search_issues",
```

Comment:
should we be using `dataset=Dataset.IssuePlatform` for the dataset and also adding search issues to the storage key in https://github.com/getsentry/sentry/blob/405923ba174dba4f71e4369796418f0b26d5f94b/src/sentry/snuba/dataset.py#L83
?

Reply:
done, thank you